_posts/blog/2018-10-11-4rios-review.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "QUEERING THE DIGITAL ARCHIVE: '4 RIOS' AND GRAPHIC NARRATION"
+modified:
+categories: blog
+excerpt: ...the unique construction of this archive calls into question institutionalized norms regarding what an archive is and how it functions.
+tags: []
+author: "Kaitlin Burns"
```

Comment:
This should be your username as you entered it into the authors.yml file (i.e. kburns073)